src/foundry/client/pixi/layers/placeables/drawings.d.ts

```diff
@@ -0,0 +1,123 @@
+export {};
+
+declare global {
+  /**
+   * The DrawingsLayer subclass of PlaceablesLayer.
+   */
+  class DrawingsLayer extends PlaceablesLayer<"Drawing"> {
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
+     */
+    static get instance(): Canvas["drawings"];
+
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
+     */
+    override options: DrawingsLayer.LayerOptions;
+
+    /**
+     * @defaultValue
+     * ```
+     * mergeObject(super.layerOptions, {
+     *   name: "drawings"
+     *   canDragCreate: true,
+     *   controllableObjects: true,
+     *   rotatableObjects: true,
+     *   zIndex: 20
+     * })
+     * ```
+     */
+    static override get layerOptions(): DrawingsLayer.LayerOptions;
+
+    static override documentName: "Drawing";
+
+    /**
+     * The named game setting which persists default drawing configuration for the User
+     */
+    static DEFAULT_CONFIG_SETTING: "defaultDrawingConfig";
+
+    /**
+     * Use an adaptive precision depending on the size of the grid
+     */
+    get gridPrecision(): 0 | 8 | 16;
+
+    override get hud(): Exclude<Canvas["hud"], undefined>["drawing"];
+
+    override get hookName(): (typeof DrawingsLayer)["name"];
```

Comment:
I still wish something like `(typeof this)["name"]` was an option. ASFIK it's not.